### PR TITLE
Accounts can be created via the frontend and users are redirected after signing up. 

### PR DESCRIPTION
**Pull request by dezok9**
Frontend and backend are now established for the first part of the signup/login user flow. 

User stories: 
- User can create an account.
- User is redirected upon login

Video showing database updating and account creation via the frontend:

Screenshot 2024-06-28 at 5 20 12 PM

https://github.com/dezok9/sync/assets/111532003/6c8b3348-a12c-4c4f-8c5a-2476ee1ab5b2

### CODE REVIEW

**Comment by ikasare** [comment]
what happens when a non existing github handle is entered?

File: frontend/src/pages/util/login.js [hunk 1]
```diff
@@ -1,20 +1,82 @@
...
+      (userInput) => userInput.replace(" ", "") === ""
+    )
+  ) {
+    if (!(email && email.includes("."))) {
+      // Invalid email address.
+      return;
+    }
+    if (password !== confirmPassword) {
+      // Passwords don't match.
+      return;
```
Comment (TsukkiGia) [inline frontend/src/pages/util/login.js:30]: Do we want to return nothing or return a boolean?

File: frontend/src/pages/util/userVerification.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      email,
+    }),
+  });
+
+  if (!status.ok) {
+    return false;
+  }
+
+  return true;
+}
```
Comment (TsukkiGia) [inline frontend/src/pages/util/userVerification.js:24]: You could return `status.ok` as opposed to checking it, and returning the same boolean

Comment (dezok9) [reply frontend/src/pages/util/userVerification.js:24]: Good catch, thank you!

File: frontend/src/pages/util/login.js (hunk 1 above)
Comment (dezok9) [reply frontend/src/pages/util/login.js:30]: Decided to return a boolean (false) for now, but when I get to focusing on CSS, I want a popup to appear.

**Pull request merged**